### PR TITLE
Amend/about pages

### PR DESCRIPTION
**Pull request by M4R1KU**
This closes nothing

*NOTE*: It is not finished but I want it to be finished asap, so I need your finished profiles and your review for the new text UI's
Also it is unclear whether we have a start page other than the about page until we make an own app for the text things.

Summary of changes:
- ​

I made sure to:
- [x] Add and / or update tests
- [x] Run the code formatter (`CTRL + ALT + L`)
- [x] Update any documentation (javadoc, domain model, erd, ...)
- [x] Follow the contribution guidelines
- [x] Update CI script and configs on Jenkins (`Jenkinsfile`, `auth0.yml`, `application.yml`, etc.)

Please review @jmesserli @needToRoll  / cc: @outcobra/developers.

### CODE REVIEW

**Comment by jmesserli** [comment]
Updated my profile in docs

**Comment by M4R1KU** [comment]
ready for review @needToRoll @jmesserli

File: frontend/src/app/app.component.html [hunk 1]
```diff
@@ -22,7 +22,7 @@
         <img routerLink="/"
              class="oc-title-logo"
              src="/assets/img/oc_logo_stroke_combined_blue_inverted.svg">
-
+        <span class="oc-title-beta">Beta</span>
```
[the remark this answers is not in the document]
Comment (M4R1KU) [reply frontend/src/app/app.component.html]: NO

[the remark this answers is not in the document]
Comment (jmesserli) [reply frontend/src/app/app.component.html]: yes, good communication

**Comment by jmesserli** [comment]
@needToRoll Can you please provide your text so we can merge this

**Comment by jmesserli** [comment]
Also there is one failing frontend test: https://ci.pegnu.cloud/blue/organizations/jenkins/outcobra%2Foutstanding-cobra/detail/PR-297/6/tests

**Pull request merged**